src/helpers.py

```diff
@@ -1019,3 +1019,14 @@ def update_ba_sheet():
         'range': f'B9:F{9 + len(player_rows)}',
         'values': player_rows
     }])
+
+def crew_count(flairing_crew: Crew, bot: 'ScoreSheetBot'):
```

Comment:
Instead of writing a new function, I think the best thing to do is to refactor crew_members slightly to use the non cached version and use the length of that instead.